modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -60,6 +60,21 @@ def validate_veteran_identifiers(require_birls: false) # rubocop:disable Metrics
         end
 
         mpi_add_response = target_veteran.mpi.add_person_proxy
+        ids = target_veteran.mpi.participant_ids
+        ids = (ids || []).push(mpi_add_response.parsed_codes[:participant_id]).uniq
```

Comment:
isn't this attempting to do the same thing as the line in `mpi_data.rb`?

Reply:
yes, but this one actually works 😄